deposit_contract/contracts/validator_registration.v.py

```diff
@@ -5,7 +5,7 @@
 SECONDS_PER_DAY: constant(uint256) = 86400
 MAX_64_BIT_VALUE: constant(uint256) = 18446744073709551615  # 2**64 - 1
 
-Deposit: event({data: bytes[528], merkle_tree_index: bytes[8]})
+Deposit: event({data: bytes[184], merkle_tree_index: bytes[8]})
```

Comment:
@JustinDrake @djrtwo @vbuterin do we want to fix the deposit data length tightly? I think allocating a bigger size is more future-proof.

Reply:
What is the gas impact either way?

Reply:
No impact if declared larger for future proof. More gas is charged only if there are more bytes to be logged.